packages/metal-component/src/Component.js

```diff
@@ -440,7 +440,9 @@ class Component extends EventEmitter {
 		const instance = new Ctor(config, false);
 
 		if (window.__METAL_DEV_TOOLS_HOOK__) {
-			window.__METAL_DEV_TOOLS_HOOK__(instance);
+			let hook = window.__METAL_DEV_TOOLS_HOOK__.addRoot ? window.__METAL_DEV_TOOLS_HOOK__.addRoot : window.__METAL_DEV_TOOLS_HOOK__;
```

Comment:
Since you're not generating a new binding on hook anywhere, that may be a const.